docker/loader.docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: '2'
 services:
   loader:
-      image: gwul/tweetsets-loader
-      #build:
-      #    context: ..
-      #    dockerfile: Dockerfile-loader
+      #image: gwul/tweetsets-loader
+      build:
+          context: ..
+          dockerfile: Dockerfile-loader
```

Comment:
@dolsysmith @lwrubel Should this change be part of the PR, or was it intended for development/testing only?

Reply:
You're right -- this should not be part of the PR. I can change it back before merging.